RetailCoder.VBE/Inspections/ObsoleteByRefModifierInspection.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Common;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.Resources;
+using Rubberduck.Inspections.Results;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+
+namespace Rubberduck.Inspections
+{
+    public sealed class ObsoleteByRefModifierInspection : InspectionBase
```

Comment:
"Obsolete" is a bit of a strong word for a keyword that's completely legit and has no alternative - how about renaming that to `RedundantByRefModifierInspection`?